activities/Markdown.activity/js/activity.js

```diff
@@ -1,13 +1,17 @@
-define(["sugar-web/activity/activity","sugar-web/datastore"], function (activity, datastore) {
+define(["sugar-web/activity/activity","webL10n","sugar-web/datastore"], function (activity, datastore) {
 
     // Manipulate the DOM only when it is ready.
     require(['domReady!'], function (doc) {
 
         // Initialize the activity.
         activity.setup();
-
+		var buttons = ["wmd-bold-button-second", "wmd-italic-button-second", "wmd-heading-button", "wmd-hr-button",
```

Comment:
this array is to translate all the buttons in https://github.com/kiridix/sugarizer/blob/master/activities/Markdown.activity/js/activity.js#L29
in order not to repeat code